app/src/main/java/org/xwiki/android/sync/utils/AccountClickListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.xwiki.android.sync.utils
+
+import android.accounts.Account
+
+interface AccountClickListener {
```

Comment:
Can be replaced via `interface AccountClickListener : (Account) -> Unit`

Reply:
Done